distributed/cli/tests/test_dask_scheduler.py

```diff
@@ -85,17 +85,17 @@ def test_dashboard(loop):
             try:
                 # All addresses should respond
                 for name in names:
-                    uri = "http://%s:%d/status/" % (name, dashboard_port)
+                    uri = f"http://{name}:{dashboard_port}/status/"
                     response = requests.get(uri)
-                    assert response.ok
+                    response.raise_for_status()
                 break
-            except Exception as f:
-                print("got error on %r: %s" % (uri, f))
+            except Exception as e:
+                print(f"Got error on {uri!r}: {e.__class__.__name__}: {e}")
+                elapsed = time() - start
+                if elapsed > 10:
+                    print(f"Timed out after {elapsed:.2f} seconds")
+                    raise
                 sleep(0.1)
-                assert time() < start + 10
-
-    with pytest.raises(Exception):
-        requests.get("http://127.0.0.1:%d/status/" % dashboard_port)
```

Comment:
Thanks @crusaderky. Overall this looks fine, though we've dropped this check here

Reply:
reinstated